app/models/match.rb

```diff
@@ -1,5 +1,36 @@
 class Match < ApplicationRecord
-  belongs_to :team_1, class_name: 'Team'
-  belongs_to :team_2, class_name: 'Team'
+  belongs_to :team_1, class_name: "Team"
+  belongs_to :team_2, class_name: "Team"
   belongs_to :game
+
+  validates :points_for_team1, presence: true
+  validates :points_for_team2, presence: true
+  validates :team_1, presence: true
```

Comment:
group all  the attributes that have the validation of presence into one validation command only